controllers/api/v1alpha1/shared_types.go

```diff
@@ -19,6 +19,8 @@ const (
 	CFRouteGUIDLabelKey      = "korifi.cloudfoundry.org/route-guid"
 	CFTaskGUIDLabelKey       = "korifi.cloudfoundry.org/task-guid"
 
+	ServiceCredentialBindingTypeLabel = "korifi.cloudfoundry.org/service-credential-binding-type"
```

Comment:
I would suggest to use a shorted name like `service-binding-type`. Even though they are really called service credential bindings by the cf v3 api spec we commonly refer to them as service bindings, as it is more concise and does not cause any confusion.